Fix-Website/Stylesheets/toc.css

```diff
@@ -0,0 +1,17 @@
+.TOC {
```

Comment:
Please review the Google style guide for HTML and CSS.
https://google.github.io/styleguide/htmlcssguide.xml

- Use a namespace, e.g. `.gtp-toc, .gtp-info`, etc.
- All lower case.